File: Bindings~/python/altunityrunner/commands/enable_logging.py
from deprecated import deprecated

from altunityrunner.commands.base_command import BaseCommand


@deprecated(version="1.6.3")
class EnableLogging(BaseCommand):
    def __init__(self, socket, request_separator, request_end, log_flag):
        super(EnableLogging, self).__init__(
            socket, request_separator, request_end)
        self.log_flag = log_flag

    def execute(self):
        if(self.log_flag):
            self.send_command('enableLogging', 'true')
        else:
            self.send_command('enableLogging', 'false')

File: Bindings~/java/src/main/java/ro/altom/altunitytester/Commands/EnableLogging.java
package ro.altom.altunitytester.Commands;

import ro.altom.altunitytester.AltBaseSettings;

@Deprecated
public class EnableLogging extends AltBaseCommand {
    public EnableLogging(AltBaseSettings altBaseSettings) {
        super(altBaseSettings);
    }

    public void Execute() {
        SendCommand("enableLogging", altBaseSettings.logEnabled.toString());
        String data = recvall();
        validateResponse("OK", data);
    }
}
